### DISCUSSION TITLE
Do not force date and time setting

### DISCUSSION THREAD

**Discussion by ruslanhirychau** [gopro/labs#370] [Feature Requests]
I may have missed it and it is already there, but it is very annoying to have to select the date and time when replacing the battery.
Let it be January 1, 1970 by default. For me it is not important when I need to quickly capture the moment.

**Comment by dnewman-gpsw** [comment]
Yes, it is annoying. Try:

![image](https://github.com/gopro/labs/assets/10856084/0335b2c3-6537-48cd-8e6f-e258f7346989)

This should bypass the date and time update, and maybe other naggy pop-ups.

**Reply by ruslanhirychau** [reply]
That's an option, but it's also time-consuming to take your phone, find the QRControl app or just the picture you suggested and scan it. I would prefer to do without this step. It would be cool to enable or disable this date and time request.

**Reply by dnewman-gpsw** [reply]
That is what this does. Scan once, and you will never be asked to set the time. This "disables the date and time request".

**Reply by ruslanhirychau** [reply]
Wow! Thanks dude!
